src/npctalk_funcs.cpp

```diff
@@ -1285,8 +1285,8 @@ void talk_function::distribute_food_auto( npc &p )
     zone_manager &mgr = zone_manager::get_manager();
     const tripoint_abs_ms &npc_abs_loc = p.get_location();
     // 3x3 square with NPC in the center, includes NPC's tile and all adjacent ones, for overflow
-    const tripoint top_left = npc_abs_loc.raw() + point{-1, -1}; // Awful hack, zones want the raw value
```

Comment:
It is still awful and notable that zone_manager::add() is expecting a raw tripoint that is actually a tripoint_abs_ms

Reply:
Added `// TODO: fix point types; Awful hack, zones want the raw value`